components/filefeed/event.tsx

```diff
@@ -4,27 +4,47 @@ type Props = {
   event: FileFeedEvent;
 };
 
+const greenEventKeywords = [
+  "added",
+  "completed",
+  "created",
+  "ready",
+  "uploaded",
+];
+const primaryEventKeywords = ["updated", "outcome-acknowledged", "scheduled"];
+const redEventKeywords = ["deleted"];
+
+const eventColors = (topic: string) => {
+  if (greenEventKeywords.some((keyword: string) => topic.includes(keyword))) {
+    return "bg-green-800";
+  } else if (
+    primaryEventKeywords.some((keyword: string) => topic.includes(keyword))
+  ) {
+    return "bg-primary";
+  } else {
+    return "bg-red-800";
+  }
+};
```

Comment:
Should the default value be the primary color? If they add more events we won't have them in the codebase, and they'll default show up as red.